python/packages/aws-lambda-sdk/serverless_aws_lambda_sdk/tests/test_internal_extension.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import annotations
```

Comment:
Ideally we should have here tests like Node.js has here: https://github.com/serverless/console/blob/fa1801ac87943becaadb6d7b6fb2ee9342124f68/node/packages/aws-lambda-sdk/test/unit/internal-extension/index.test.js

Those tests set up all needed AWS Lambda environment variables, run internal extension with handler path pointing to pre-prepared lambda fixtures, then run obtained handler and confirm that they operate as expected.

At this stage, we should just prepare a few basic fixtures (success and error case?) and run them through machinery and confirm we have the expected outcome.

In the next task, when we will address trace spans configuration and the trace report will be written, we will also read that payload and confirm it contains the expected information.

Aside of that, as I announced we will also have integration tests which will be configured in Node.js and through real AWS lambdas will confirm the same.
As I think of it, in the scope of this task (as there's no instrumentation) it's probably too early for integration tests (which, in this case, should skip on confirming trace payloads, while it's their main point), so let's agree we will add them once tracing is configured (it'll save us back and forth work). For now just please manually confirm that in the AWS environment published basic cases of lambda work the same with the wrapping as they do without.

Reply:
>  and run them through machinery and confirm we have the expected outcome.

> For now just please manually confirm that in the AWS environment published basic cases of lambda work the same with the wrapping as they do without.

When you say this, do you mean via tests or literally going into AWS Lambda and running it?

Reply:
 > When you say this, do you mean via tests or literally going into AWS Lambda and running it?
 
 I suggested that you test if this wrapping works without issues in AWS environment, manually, as at this stage we won't have integration test configured for it.